gulp-tasks/watch.js

```diff
@@ -1,21 +1,27 @@
 const gulp = require('gulp');
 
+const build = require('./build');
+
 const logHelper = require('../infra/utils/log-helper');
 
-gulp.task('watch', gulp.series(
-  'build',
+const watch = gulp.series(
+  build,
   () => {
     gulp.watch(
       [
         `packages/${global.packageOrStar}/**/*.mjs`,
         '!packages/**/_version.mjs',
         '!packages/**/build/**/*',
         '!packages/**/node_modules/**/*',
-      ], gulp.series('build'),
+      ], build,
     )
+    // GULP: why is this here?
     .on('error', () => {})
     .on('change', function(path, stats) {
       logHelper.log(`gulp.watch() is running due to a change in '${path}'`);
     });
   }
-));
+);
+watch.displayName = 'watch';
```

Comment:
Is this needed?

Reply:
Nope, this would be better served as a named function.  I started with that but switched it to an arrow function with `displayName` to keep things consistent.